src/nano_contracts/deserializer.ts

```diff
@@ -328,13 +339,15 @@ class Deserializer {
    * It does not support chained container types, meaning Tuple[Dict[str,str]] should not happen.
    *
    * @param buf Value to deserialize
-   * @param type Comma separated types, e.g. `str,int,VarInt`
+   * @param typeArr Comma separated types, e.g. `str,int,VarInt`
```

Comment:
question: If I understand correctly, this is an actual array, not a comma separated string value. If so, the docstring should reflect this.

Reply:
Fixed